axelor-account/src/main/resources/i18n/messages_fr.csv

```diff
@@ -3384,6 +3416,8 @@
 "Warning : The following invoice terms have been paid or partially paid since the payment session was created and thus won't be considered during validation. Do you wish to continue ?","Attention : Les échéances suivantes ont été payées ou partiellement payées et ne vont pas être prises en compte durant la validation. Souhaitez-vous continuer ?",,
 "Warning : This will import move lines even if they have already been imported through this feature. Be careful not to import the same elements several times.","Attention, les lignes d'écriture seront importées même si elles ont déjà fait l'objet d'un import. Veillez à ne pas importer les mêmes éléments plusieurs fois.",,
 "Warning, existing lines will be deleted, do you wish to continue?","Attention, les lignes existantes vont être supprimées, voulez-vous continuer ?",,
+"Warning, the option of lettering proposal is not activated, the realized reconcile groups will be definitively validated without the possibility of checking them beforehand.","Attention, l'option de proposition de lettrage n'est pas activée, les lettrages réalisés seront validés définitivement sans possibilité de les contrôler au préalable.",,
+"Warning, there are still reconcile group proposals pending that correspond to the filters you have entered. Do you want to continue ?","Attention, il reste des propositions de lettrage en attente correspondant aux filtres que vous avez renseignés. Voulez-vous continuer ?",,

```

Comment:
We have to change this message because it doesn't make sense, the user doesn't have a choice, we don't allow him to continue. 
Change it to: "Warning, there are still reconcile group proposals pending that correspond to the filters you have entered."
Translation ""Attention, il reste des propositions de lettrage en attente correspondant aux filtres que vous avez renseignés."